enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -110,6 +116,24 @@ Although cluster infra resources already do a best effort validation on the user
 a user could provide non-functional proxy settings/certs. This would be propagated to their Windows nodes and workloads,
 taking down existing application connectivity and preventing new Windows nodes from being bootstrapped.
 
+To enable global HTTP proxy by default on Windows nodes with cluster-wide proxy for all PowerShell's sessions, 
```

Comment:
@jrvaldes this section is for risk and mitigation, the risk needs to be explained first followed by its mitigation.

Reply:
I think the mitigation implies the risk but it might be useful to call it out directly:

`If users use Windows images with PowerShell as the default shell, then there is a risk that outbound traffic from the PowerShell CLI wouldn't go through the cluster-wide proxy by default. Although this does not affect WMCO/OpenShift's view of the Node, this is different on-instance behavior than an admin would see if they were using CMD Prompt. To mitigate this...`

Reply:
Addressed.